mteb/tasks/Retrieval/eng/LEMBSummScreenFDRetrieval.py

```diff
@@ -0,0 +1,84 @@
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class LEMBSummScreenFDRetrieval(AbsTaskRetrieval):
+    _EVAL_SPLIT = "validation"
+
+    metadata = TaskMetadata(
+        name="LEMBSummScreenFDRetrieval",
+        dataset={
+            "path": "dwzhu/LongEmbed",
+            "revision": "6e346642246bfb4928c560ee08640dc84d074e8c",
+            "name": "summ_screen_fd",
+        },
+        reference="https://huggingface.co/datasets/dwzhu/LongEmbed",
+        description=("summ_screen_fd subset of dwzhu/LongEmbed dataset."),
+        type="Retrieval",
+        category="s2p",
+        eval_splits=[_EVAL_SPLIT],
+        eval_langs=["eng-Latn"],
+        main_score="ndcg_at_10",
+        date=None,
+        form=None,
+        domains=None,
+        task_subtypes=None,
+        license=None,
+        socioeconomic_status=None,
+        annotations_creators=None,
+        dialect=None,
+        text_creation=None,
+        bibtex_citation=None,
+        n_samples=None,
+        avg_character_length=None,
+    )
+
+    # @property
+    # def description(self):
+    #     return {
+    #         'name': 'LEMBSummScreenFDRetrieval',
+    #         'dataset_name': 'dwzhu/LongEmbed',
+    #         'subset': 'summ_screen_fd',
+    #         'reference': 'https://huggingface.co/datasets/dwzhu/LongEmbed',
+    #         "description": (
+    #             "summ_screen_fd subset of dwzhu/LongEmbed dataset. "
+    #         ),
+    #         "type": "Retrieval",
+    #         "category": "s2p",
+    #         "eval_splits": ["validation"],
+    #         "eval_langs": ["en"],
+    #         "main_score": "ndcg_at_10",
+    #     }
```

Comment:
```suggestion
```

Reply:
thanks for this, I have removed these dead codes